{
  "uid" : "7b20b8eea946c420",
  "name" : "test_02_add_member[已经存在的会员号新增-92772-19827162581-301-数据写入异常]",
  "fullName" : "Case.test_02_member#test_02_add_member",
  "historyId" : "29a2e382279fa4e49ee3aaac125461fd",
  "time" : {
    "start" : 1723540741601,
    "stop" : 1723540741654,
    "duration" : 53
  },
  "status" : "failed",
  "statusMessage" : "pytest_assume.plugin.FailedAssumption: \n2 Failed Assumptions:\n\nCase\\test_02_member.py:17: AssumptionFailure\n>>\tpytest.assume(GetKeywords.get_keyword(result,\"code\") == int(code))\nAssertionError: assert False\n\nCase\\test_02_member.py:18: AssumptionFailure\n>>\tpytest.assume(GetKeywords.get_keyword(result,\"message\") == message)\nAssertionError: assert False",
  "statusTrace" : "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               2 Failed Assumptions:\nE               \nE               Case\\test_02_member.py:17: AssumptionFailure\nE               >>\tpytest.assume(GetKeywords.get_keyword(result,\"code\") == int(code))\nE               AssertionError: assert False\nE               \nE               Case\\test_02_member.py:18: AssumptionFailure\nE               >>\tpytest.assume(GetKeywords.get_keyword(result,\"message\") == message)\nE               AssertionError: assert False\n\nD:\\py3\\lib\\site-packages\\six.py:718: FailedAssumption",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "member",
    "time" : {
      "start" : 1723540741576,
      "stop" : 1723540741601,
      "duration" : 25
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1723540740919,
      "stop" : 1723540741037,
      "duration" : 118
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "Case"
  }, {
    "name" : "suite",
    "value" : "test_02_member"
  }, {
    "name" : "host",
    "value" : "LAPTOP-J99CE7C3"
  }, {
    "name" : "thread",
    "value" : "4892-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "Case.test_02_member"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "code",
    "value" : "'301'"
  }, {
    "name" : "message",
    "value" : "'数据写入异常'"
  }, {
    "name" : "supplier",
    "value" : "'92772'"
  }, {
    "name" : "telephone",
    "value" : "'19827162581'"
  }, {
    "name" : "title",
    "value" : "'已经存在的会员号新增'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "7b20b8eea946c420.json",
  "parameterValues" : [ "'301'", "'数据写入异常'", "'92772'", "'19827162581'", "'已经存在的会员号新增'" ]
}